enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/LdapRealm.java

```diff
@@ -122,6 +133,8 @@ protected AuthorizationInfo queryForAuthorizationInfo( PrincipalCollection princ
                 if ( authorizationInfo == null )
                 {
                     // TODO: Do a new LDAP search? But we need to cache the credentials for that...
+                    // Or we need the resulting failure message to the client to contain some status
+                    // so that the client can react by resending the auth token.
```

Comment:
Are these ideas cardified somewhere or do they live here only? When will we solve these?